htdocs/libraries/icms/core/DataFilter.php

```diff
@@ -164,7 +164,7 @@ static public function htmlEntities($text) {
 	 * Add slashes to the text if magic_quotes_gpc is turned off.
 	 *
 	 */
-	static public function addSlashes(string $text, string $param) {
+	static public function addSlashes(string $text, string $param = null ) {
 		if (!get_magic_quotes_gpc()) {
 			$text = addslashes($text, $param);
```

Comment:
https://www.php.net/manual/en/function.addslashes.php seems doesn't have 2nd parameter. so, maybe we can just  remove $param from addSlashes ?

Reply:
I forgot the other change : instead of calling addslashes, I now call addcslashes, which has the possibility to define for which data specifically you want to add slashes. That would be needed in order to define slashes for % in mysql